Assets/Shaders/CrossHairShotgun.hlsl

```diff
@@ -37,5 +37,6 @@ float shotgun_float(float2 pos)
 
 void CrossHairShotgun_float(float2 UV, float crossSize, float circleRadius, float hitMarkerRadius, out float Distance)
 {
-	Distance = min(min(shotgun_float(UV * circleRadius), crossSDFShotgun_float(UV * crossSize, 1.)), sdRoundedXShotgun_float(UV, hitMarkerRadius, 0.1));
+    UV *= 0.9;
```

Comment:
Huh, so you just made the whole thing smaller?
What I wanted to point out was the jarring difference in thinness between this cross
![image](https://github.com/user-attachments/assets/0846e74d-7a66-4cc7-9b9a-1bdf8ba2114d)
and this cross
![image](https://github.com/user-attachments/assets/c9e58f30-2939-409c-a8bd-ef57e9772c0e)
which has become even greater now that they differ this much in length.

Reply:
It was both, the crosshair became thinner, shotgun specifically was additionaly shrunk.
The observed thickness difference stems from the center being rounded towards the arms of the cross, thus seeming thicker than they actually are for crosshairs with shorter "hands"

Hat now has an even smaller non-reactive cross instead, with the punch bein applied to the hatSDF instead